samples/Server/src/routes/addUserToRoom.ts

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as express from 'express';
+import { getResourceConnectionString } from '../lib/envHelper';
+import { RoomsClient, RoomParticipant, Role } from '@azure/communication-rooms';
+
+const router = express.Router();
+interface AddUserToRoomParam {
+  userId: string;
+  roomId: string;
+  role: string;
+}
+
+/**
+ * route: /addUserToRoom
+ *
+ * purpose: Add the user to the room with given roomId and role.
+ *
+ * @param userId: id of the user as string
+ * @param roomId: roomId to add user as string
+ * @param role: role of user as string
+ */
+
+router.post('/', async function (req, res, next) {
+  // create RoomsClient
```

Comment:
nit: this comment doesn't add value as it is just stating the code below it (can remove it)